tests/go.rs

```diff
@@ -1,11 +1,13 @@
+#[cfg(test)]
 mod tests {
-    use std::env;
-
     use tinkoff_invest_api::{TIResult, TinkoffInvestService};
 
     #[tokio::test]
     async fn test_get_accounts() -> TIResult<()> {
-        let service = TinkoffInvestService::new(env::var("TINKOFF_SDK_TESTING_ENV").unwrap());
+        let token =
+        "t.gktnxozBwY1vxHe_ybs_IwE-vaFK2_4Gt7DGBe0uloap5lsxpd4_nVVNr42lQh_DHbdG3j1fGZcZMFX_B2uwRw"
```

Comment:
@azoyan you need to keep your token as a secret... Please revert + refresh it with Tinkoff.

Reply:
it is done.